RestaurantReservation.Db/Repositories/CustomerRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RestaurantReservation.Db.DataModels;
+
+namespace RestaurantReservation.Db.Repositories
+{
+    public class CustomerRepository
+    {
+        private readonly RestaurantReservationDbContext _context;
+
+        public CustomerRepository(RestaurantReservationDbContext context)
+        {
+            _context = context;
+        }
+
+        public void CreateCustomer(Customer customer)
+        {
+            _context.Customers.Add(customer);
+            _context.SaveChanges();
+        }
+
+        public void UpdateCustomer(Customer customer)
+        {
+            var existingCustomer = _context.Customers.Find(customer.CustomerId);
+            if (existingCustomer != null)
+            {
+                existingCustomer.FirstName = customer.FirstName;
+                existingCustomer.LastName = customer.LastName;
+                existingCustomer.Email = customer.Email;
+                existingCustomer.PhoneNumber = customer.PhoneNumber;
+                existingCustomer.Reservations = customer.Reservations;
+                _context.SaveChanges();
+            }
```

Comment:
Since you're updating all the properties of the customer, you can simplify the code by using:
```

_context.Customers.Update(customer);
_context.SaveChanges();
```
Entity Framework tracks changes automatically, so there’s no need to manually retrieve and update each property. This approach is cleaner, more efficient, and ensures that all properties are updated without explicitly setting each one